packages/shared/src/store/assets/fungibleTokensSlice.ts

```diff
@@ -46,6 +47,12 @@ const fungibleTokensSlice = createSlice({
       .addCase(syncVerifiedFungibleTokens.pending, (state) => {
         state.loadingVerified = true
       })
+      .addCase(syncFungibleTokensInfo.pending, (state) => {
+        state.loadingUnverified = true
+      })
+      .addCase(syncUnknownTokensInfo.pending, (state) => {
```

Comment:
Nit: Should we rename `syncUnknownTokensInfo` into something like `syncTokenTypes` to better match the `loadingTokenTypes` bool?

Reply:
`syncUnknownTokensInfo` is doing 3 things:
1. Gets the types
2. Calls `syncFungibleTokensInfo`
3. Calls `syncNFTsInfo`